src/RIO/Text.hs

```diff
@@ -1,20 +1,151 @@
+{-# LANGUAGE MagicHash #-}
```

Comment:
I'm surprised this is necessary.

Reply:
Me too, but exporting `unpackCString#` without -XMagicHash is a syntax error :/

Reply:
Doh, I missed that one!